cmd/kogito/command/deploy/deploy_service_test.go

```diff
@@ -331,3 +331,71 @@ func Test_WithManagementConsole(t *testing.T) {
 	assert.Contains(t, lines, "successfully created")
 	assert.Contains(t, lines, mgmtConsoleURI)
 }
+
+func Test_DeployCmd_WithCustomArtifactDetails(t *testing.T) {
+	ns := t.Name()
+	groupId := "com.mycompany"
+	artifactId := "testproject"
+	version := "2.0-SNAPSHOT"
+	fileLocation, _ := url.Parse("https://raw.githubusercontent.com/kiegroup/kogito-examples/master/dmn-quarkus-example/src/main/resources/Traffic Violation.dmn")
```

Comment:
Instead, please bring the file to the `testdata` directory and use it from there.

Reply:
actual fileLocation never refered while running BATS, still I replace to testdata location